extensions/scarb-snforge-test-collector/src/compilation/test_collector/config.rs

```diff
@@ -140,6 +145,23 @@ pub fn forge_try_extract_test_config(
         None
     };
 
+    let max_steps = if let Some(attr) = max_steps_attr {
+        if attr.args.is_empty() {
+            None
+        } else {
```

Comment:
Just now realised this approach is just wrong (I know it was used above, it has to be fixed too, sooner or later). If there are no args here we should just throw an error, it is incorrect to have sth like `#[max_steps]` there (same with `#[fork]` and `#[fuzzer]`). I would leave it as it is and create an issue for that.